libraries/abstractions/common_io/include/iot_spi.h

```diff
@@ -155,86 +163,132 @@ void iot_spi_set_callback( IotSPIHandle_t const pxSPIPeripheral,
 /**
  * @brief Configures the SPI port with user configuration.
  *
- * The application should call this function to configure the SPI driver
- * with values in the structure IotSPIMasterConfig_t. The application also needs to
- * specify the request which is as per enum IotSPIIoctlRequest_t.
+ *
+ * @note eSPISetMasterConfig is expected only called once at beginning.
```

Comment:
nit: This can be called at any time (but bus must be idle for setting the master config)

Reply:
ok, I will rephrase.